### ISSUE TITLE
Deprecated warning when installing SDK

### ISSUE THREAD

**Issue by octonato** [lightbend/akkaserverless-javascript-sdk#32]
We get a few deprecated warnings when running

```shell
npm install @lightbend/akkaserverless-javascript-sdk@0.7.0-beta.9
```

```node
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm WARN deprecated grpc@1.24.10: This library will not receive further updates other than security fixes. We recommend using @grpc/grpc-js instead.
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
```

**Comment by johanandren** [comment]
I wonder if these warnings has been fixed by the switch to grpc-tools in #38 @andreaTP ?

**Comment by andreaTP** [comment]
Unfortunately, the switch to `grpc-js` resolves just one of the warnings.

> request@2.88.2

is still there but doesn't spill to user-land. (e.g. is used to download protoc)

> har-validator
> uuid

are transitive dependencies of the first.

**Comment by andreaTP** [comment]
fixed in `0.7.0-beta.12`
